tests/test_shiny_dashboards.py

```diff
@@ -12,6 +12,15 @@ def setUpClass(cls) -> None:
             cls.credentials = json.load(f)
         cls.api = create_api(verify=True, **cls.credentials)
 
+    @classmethod
+    def tearDownClass(cls) -> None:
+        id_ = cls.api.get_shiny_dashboard_id('pylana-shiny-dashboard')
+        cls.api.delete_dashboard(id_)
+
+        shiny_dashboard_id = cls.api.get_shiny_dashboard_id('incident-test-shiny-dashboard')
+        cls.api.unshare_dashboard(shiny_dashboard_id,
+                                  [cls.api.user.organization_id]).json()
```

Comment:
t is good to denote discarded return values with an underscore `_ = ...`. Also, we don't need the `.json()` here.